cmd/vic-machine/validate/validator.go

```diff
@@ -350,6 +354,62 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *metadat
 	}
 }
 
+func (v *Validator) GetVCH(input *data.Data) (*object.ResourcePool, error) {
```

Comment:
Validator is not the right place for this function - the only reason to have this in validator is because of the compute resource lookup code which should be moved under pkg/vsphere/ at some point anyway.
Validator could and should expose an IsAppliance(object.VirtualMachine) function that will check assertions about the VM config, but that's it.

This is the some of the same function we'll need for `ls` and `inspect` so let's get it in the right place. I think that's in lib/install/management. These functions can locate an appliance VM if possible, then pass that to validator for checking:

```
NewFromID(moref string) (*management.VCH, error)
NewFromCompute(path string) (*management.VCH, err) // specified by compute resource path
NewFromPath(path string) (*management.VCH, err) // specified by folder path when we add folder support.
```

We can then add functions to management.VCH for:
* inspect
* delete
* update